src/main/java/io/github/arrayv/main/RunSort.java

```diff
@@ -197,27 +198,28 @@ public void run() {
                     }
 
                     if (goAhead) {
-                        if (sort.getRunSortName().equals("In-Place LSD Radix")) {
+                        Sort sortInstance = sort.getFreshInstance();
```

Comment:
Ok

Reply:
What would you propose?

Reply:
`SortInfo sortInfo` and `Sort sort`

Reply:
are we keeping `Sort sortInstance`? I feel `Sort sort` is more consistent, though I don't remember how it's been done elsewhere

Reply:
`sortInstance` is a clearer name.

Reply:
Is it really? The class is called `Sort`, and I don't see anything else it could be confused with (notwithstanding other possible instances of the confusing labeling)

Reply:
This is way too nitpicky. What I call a variable name in my own code that I contribute is my business, as long as it's not completely ridiculous. When you contribute something, name your variables how you like.

Reply:
I mean, you are asking me to review this PR, and the reason why is not just to ensure that it works, but to make sure the code is quality and considerate of future changes. Variable naming is one of those things, and although it may sound nitpicky, I think sticking to a consistent naming scheme is simple and a good habit.

But on the other hand, I think I now see how sorts are being restructured around `SortInfo`, so in that case I get the naming. If that's how you see it then I think that's nice. Though I'm now wondering about how all the sort class stuff could be named, and what a "`Sort`" truly is 🤔

Reply:
My reasoning for asking you to review is to look for mistakes and check minor code style things, not to nitpick each and every style choice.

Reply:
fair enough, and bad phrasing on my part